src/main/resources/templates/aeolus/ruby/default.sh

```
@@ -1,5 +1,11 @@
 #!/usr/bin/env bash
 set -e
+export AEOLUS_INITIAL_DIRECTORY=${PWD}
+install_dependencies () {
+  echo '⚙️ executing install_dependencies'
+  cd "${testWorkingDirectory}"
```

Shellcheck (0.10.0)

[warning] testWorkingDirectory is referenced but not assigned.

(SC2154)